### ISSUE TITLE
TypeError when making doc

### ISSUE THREAD

**Issue by jdreo** [saezlab/BioCypher#74]
Trying to build the doc ends on a TypeError.

# Steps

1. install from git using poetry, as advised
2. poetry shell
3. cd docs
4. make html

Error:
```
Sphinx v5.3.0 en cours d'exécution
INFO -- This is BioCypher v0.3.4.
INFO -- Logging into `biocypher-log/biocypher-20221110-091324.log`.

Configuration error:
Votre fichier de configuration comporte une erreur de programmation : 

Traceback (most recent call last):
  File "/home/nojhan/.cache/pypoetry/virtualenvs/biocypher-MNYv4O6r-py3.8/lib/python3.8/site-packages/sphinx/config.py", line 350, in eval_config_file
    exec(code, namespace)
  File "/home/nojhan/code/BioCypher/docs/conf.py", line 18, in <module>
    import biocypher  # noqa: E402
  File "/home/nojhan/code/BioCypher/biocypher/__init__.py", line 26, in <module>
    from ._driver import Driver
  File "/home/nojhan/code/BioCypher/biocypher/_driver.py", line 37, in <module>
    from ._translate import Translator, BiolinkAdapter
  File "/home/nojhan/code/BioCypher/biocypher/_translate.py", line 61, in <module>
    class BiolinkAdapter:
  File "/home/nojhan/code/BioCypher/biocypher/_translate.py", line 562, in BiolinkAdapter
    def trim_biolink_ancestry(ancestry: list[str]) -> list[str]:
TypeError: 'type' object is not subscriptable

make: *** [Makefile:20 : html] Erreur 2
```
Log file:
```
2022-11-10 09:13:24,371	INFO	module:_logger
This is BioCypher v0.3.4.
2022-11-10 09:13:24,373	INFO	module:_logger
Logging into `biocypher-log/biocypher-20221110-091324.log`.
2022-11-10 09:13:24,373	DEBUG	module:_driver
Loading module biocypher._driver.
2022-11-10 09:13:24,491	DEBUG	module:_write
Loading module biocypher._write.
2022-11-10 09:13:24,492	DEBUG	module:_create
Loading module biocypher._create.
2022-11-10 09:13:24,494	DEBUG	module:_translate
Loading module biocypher._translate.
```

# Configuration

- BioCypher cloned from Git at 132dfca
- Python: 3.8.10
- sphinx-build: 5.3.0
- make: 4.2.1
- LSB Version:	core-11.1.0ubuntu2-noarch:security-11.1.0ubuntu2-noarch
- Distributor ID:	Ubuntu
- Description:	Ubuntu 20.04.5 LTS
- Release:	20.04
- Codename:	focal

**Comment by jdreo** [comment]
The same error occurs when trying to run the tutorial with `cd tutorial ; python3 01_basic_import.py`.

**Comment by slobentanzer** [comment]
Hi @jdreo,
thanks for the report. I can't reproduce this behaviour, although I have some other issues when I try to switch around python versions. Is the BioCypher root directory in your `sys.path` when executing? What happens if you call, from the root of the project, `python tutorial/01_basic_import.py`?

**Comment by slobentanzer** [comment]
Thanks for the report! The type hinting we are using is not supported in `python 3.8`. I updated the `pyproject.toml` to `python = "^3.9"` in d320fc88f021c5de1ec508f81f05a376c54ed4b2. Hope that fixes it!

**Comment by jdreo** [comment]
Works with the fix and python 3.9.
